sklearn/tree/_tree.pyx

```diff
@@ -1477,29 +1496,23 @@ cdef _cost_complexity_prune(unsigned char[:] leaves_in_subtree, # OUT
             r_node[i] = (
                 weighted_n_node_samples[i] * impurity[i] / total_sum_weights)
 
-        # Push root node, using StackRecord.start as node id
-        rc = stack.push(0, 0, 0, -1, 0, 0, 0)
+        ccp_stack.push({"node_idx": 0, "parent": -1})
```

Comment:
I don't know if this is worth mentioning that other fields are zero-initialized and this is a defined behaviour?